milli/src/search/facet/filter.rs

```diff
@@ -342,12 +340,12 @@ impl<'a> Filter<'a> {
         match &self.condition {
             FilterCondition::Condition { fid, op } => {
                 let filterable_fields = index.filterable_fields(rtxn)?;
-                if filterable_fields.contains(&fid.to_lowercase()) {
+                if filterable_fields.contains(fid.value()) {
```

Comment:
Don't you implement `Deref` on that type? Do you need to call `value()` here?

Reply:
Yes but `Deref` returns the « raw » value and I think here it's better to get the real value in it (with the unescaped quote if there was any)

Reply:
Ho, yeah. Not sure if Deref should return the raw or escaped value here :) But that'll part of another PR 😄